Lists T2 connections in README

adkron opened the pull request:
The T2 connections are not labeled the same as the T1. The module works great, but the
connections needed an update. I used the current T2 port documentation and tested the
module. Everything looks great.

Amos King @adkron amos@binarynoggin.com

adkron committed:
Lists T2 connections in README

The T2 connections are not labeled the same as the T1. The module works great, but the
connections needed an update. I used the current T2 port documentation and tested the
module. Everything looks great.

Amos King @adkron <amos@binarynoggin.com>

Frijol commented:
Thanks for this! One improvement: it would be nice to note that the connections are exactly the same, just the labeling is different.

Also, for your reference: flip over Tessel & read the labeling on that side, should be more similar to T1

adkron committed:
Clarifies the port differences between T1 and T2

Since the ports are in the same location as the original it would
be good for users to know that their modules can still connect.
I also noted that the connections are labeled with the older labels
on the underside of the T2.

Amos King @adkron <amos@binarynoggin.com>

adkron commented:
@Frijol I updated it.

Frijol commented:
Perfect, thanks!
